Error: Option 'declaration' cannot be specified with option 'isolatedModules'

<!-- This template is for bugs relating to Ember.js typescript support & infrastructure.
     Please fill out all of the required information below -->

### Please paste the output of `ember -v` here

```
ember-cli: 3.6.0
node: 10.13.0
os: darwin x64
```

### Please paste the output of `tsc -v` here

```
Version 3.2.2
```

### Please paste your `tconfig.json` and `tslint.json` or `eslint.json` (if applicable) below


<details><summary><b>My tsconfig.json</b></summary><pre>
{
  "compilerOptions": {
    "target": "es2017",
    "allowJs": true,
    "moduleResolution": "node",
    "isolatedModules": true,
    "allowSyntheticDefaultImports": true,
    "noImplicitAny": true,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "strictNullChecks": true,
    "strictPropertyInitialization": true,
    "noFallthroughCasesInSwitch": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noEmitOnError": false,
    "noEmit": true,
    "inlineSourceMap": true,
    "inlineSources": true,
    "baseUrl": ".",
    "module": "es6",
    "experimentalDecorators": true,
    "paths": {
      "dummy/tests/*": ["tests/*"],
      "dummy/*": ["tests/dummy/app/*", "app/*"],
      "ember-drafts": ["addon"],
      "ember-drafts/*": ["addon/*"],
      "ember-drafts/test-support": ["addon-test-support"],
      "ember-drafts/test-support/*": ["addon-test-support/*"],
      "*": ["types/*"]
    }
  },
  "include": [
    "app/**/*",
    "addon/**/*",
    "tests/**/*",
    "types/**/*",
    "test-support/**/*",
    "addon-test-support/**/*"
  ]
}
</pre></details>


### What are instructions we can follow to reproduce the issue?
```sh
ember addon sample; cd ./sample # Create a new ember app
ember install -S ember-cli-typescript@next # Set up typescript support
ember ts:precompile
```

### Now about that bug. What did you expect to see?

I expected the default configuration to be compile-able.

### What happened instead?

I got the following TypeScript error:

> tsconfig.json(6,5): error TS5053: Option 'declaration' cannot be specified with option 'isolatedModules'.

--- a/ts/lib/commands/precompile.ts
+++ b/ts/lib/commands/precompile.ts
@@ -28,6 +28,7 @@ export default command({
       '--allowJs', 'false',
       '--noEmit', 'false',
       '--rootDir', rootDir || this.project.root,
+      '--isolatedModules', 'false',
       '--declaration',
       '--declarationDir', outDir,
       '--emitDeclarationOnly',

